### PR TITLE
Add Kailun Yang (Hunan University)

### PR DESCRIPTION
**Pull request by elnino9ykl**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-k.csv [hunk 1]
```diff
@@ -74,12 +74,14 @@ Kaigui Bian,Peking University,http://net.pku.edu.cn/~bkg,P8WKqa4AAAAJ
...
 Kaili Müürisep,University of Tartu,http://kodu.ut.ee/~kaili,BSsD0vgAAAAJ
 Kaili Zhao,BUPT,https://zkl20061823.github.io,kRbj_0kAAAAJ
+Kailun Yang 0001,Hunan University,https://yangkailun.com,pKFqWhgAAAAJ
 Kaiming Shen,CUHK (SZ),https://myweb.cuhk.edu.cn/shenkaiming,ZXex-IkAAAAJ
 Kaiqi Huang,Chinese Academy of Sciences,http://people.ucas.ac.cn/~huangkaiqi?language=en,caQ-OmYAAAAJ
 Kaisa Väänänen,Tampere University,https://research.tuni.fi/kaisa-vaananen,mt41HYQAAAAJ
 Kaishun Wu,Shenzhen University,http://www.kaishunwu.com/index.html,Vt-q3E4AAAAJ
 Kaitai Liang,TU Delft,https://www.tudelft.nl/ewi/over-de-faculteit/afdelingen/intelligent-systems/cybersecurity/computational-privacy/people/kaitai-liang,_qHKRH4AAAAJ
 Kaito Fujii,National Institute of Informatics,https://www.nii.ac.jp/en/faculty/informatics/fujii_kaito/,RcFjdYwAAAAJ
+Kaiwei Wang,Zhejiang University,http://wangkaiwei.org/indexeg.html,B6xWNvgAAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv]: Not in CS. Please clarify how they meet the inclusion criteria.

File: csrankings-k.csv [hunk 2]
```diff
@@ -74,12 +74,14 @@ Kaigui Bian,Peking University,http://net.pku.edu.cn/~bkg,P8WKqa4AAAAJ
 Kaihuai Qin,Tsinghua University,http://www.tsinghua.edu.cn/publish/cs/4616/2011/20110907090317797876810/20110907090317797876810_.html,NOSCHOLARPAGE
 Kaili Müürisep,University of Tartu,http://kodu.ut.ee/~kaili,BSsD0vgAAAAJ
 Kaili Zhao,BUPT,https://zkl20061823.github.io,kRbj_0kAAAAJ
+Kailun Yang 0001,Hunan University,https://yangkailun.com,pKFqWhgAAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv]: This looks like an engineering department. Please clarify how they meet the inclusion criteria (ability to solely advise a PhD student for a PhD _in Computer Science_).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-k.csv (hunk 1 above)
Comment (elnino9ykl) [reply csrankings-k.csv]: Prof. Kaiwei Wang is a full professor, who advises Ph.D. students in Information Sensing and Instrumentation, a combination of computer science (informatics) and optics. So Prof. Kaiwei Wang meets the inclusion criteria.

File: csrankings-k.csv (hunk 2 above)
Comment (elnino9ykl) [reply csrankings-k.csv]: Prof. Kailun Yang is an associate professor at the School of Robotics (a Department for Robotics Science and Engineering) at Hunan University. He can solely advise Ph.D. students in Robotics and Computer Science.

File: csrankings-k.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings-k.csv]: That is not an accurate reading of the requirements. They must be able to solely advise PhD students for a degree _in Computer Science_.

File: csrankings-k.csv (hunk 2 above)
Comment (emeryberger) [reply csrankings-k.csv]: Please point to a web page listing the CS faculty so I can verify this.

Comment (elnino9ykl) [reply csrankings-k.csv]: A list of professors of robotics:
http://robotics.hnu.edu.cn/info/1003/1196.htm

The webpage of Prof. Kailun Yang:
http://robotics.hnu.edu.cn/info/1071/2119.htm

Please verify. Thanks.

File: csrankings-k.csv (hunk 1 above)
Comment (elnino9ykl) [reply csrankings-k.csv]: Prof. Kaiwei Wang solely advises Ph.D. students for a degree in Information Sensing and Instrumentation.
https://person.zju.edu.cn/en/0009214

If this does not meet the inclusion criteria, then please leave Prof. Kaiwei Wang out and only include Prof. Kailun Yang.

Thanks.

File: csrankings-k.csv (hunk 2 above)
Comment (emeryberger) [reply csrankings-k.csv]: This doesn't help clarify the matter. I need to see _CS_ faculty, or indication of an affiliation / cross-appointment.

Comment (elnino9ykl) [reply csrankings-k.csv]: http://robotics.hnu.edu.cn/info/1010/1357.htm

This page clarifies that the School of Robotics at Hunan University intertwines computer science and technology and other related academic disciplines. The professors can also advise Ph.D. students in computer science. Does this help?

Comment (emeryberger) [reply csrankings-k.csv]: The only question is whether they can _solely_ advise PhD students for a PhD _in Computer Science_.

Comment (elnino9ykl) [reply csrankings-k.csv]: Yes. They can. If this does not help, then close this request without merging.

File: csrankings-k.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings-k.csv]: Should be removed. Please remove this entry.

File: csrankings-k.csv (hunk 2 above)
Comment (emeryberger) [reply csrankings-k.csv]: Happy to merge your entry since you meet the inclusion criteria. Please remove the other faculty member. Thanks.

Comment (elnino9ykl) [reply csrankings-k.csv]: Done. Thanks.

File: csrankings-k.csv (hunk 1 above)
Comment (elnino9ykl) [reply csrankings-k.csv]: It has been removed. Thanks.

File: csrankings-k.csv (hunk 2 above)
Comment (elnino9ykl) [reply csrankings-k.csv]: I only changed one line and please verify. The entry of Kaiwei Wang is removed and Kailun Yang 0001 is remained.

Comment (elnino9ykl) [reply csrankings-k.csv]: I created another request here. Thanks.

https://github.com/emeryberger/CSrankings/pull/5939

**Pull request closed without merging**